src/Cli/Microsoft.TemplateEngine.Cli/Commands/list/BaseListCommand.cs

```diff
@@ -46,6 +47,13 @@ internal BaseListCommand(
             Arity = new ArgumentArity(0, 1)
         };
 
+        internal static CliOption<FormatOptions> FormatOption { get; } = new("--format")
+        {
+            AllowMultipleArgumentsPerToken = true,
+            Description = SymbolStrings.ListCommand_Option_Format,
+            Arity = new ArgumentArity(1, 1),
```

Comment:
nit: consider using `ArgumentArity.ExactlyOne` here since it's the same as `ArgumentArity(1,1)`.

Reply:
Done. Thanks for the suggestion!